frontend/src/metabase/nav/components/search/SearchBar/CommandPaletteTrigger.tsx

```diff
@@ -0,0 +1,40 @@
+import type React from "react";
+import { t } from "ttag";
+
+import { isMac } from "metabase/lib/browser";
+import { color } from "metabase/lib/colors";
+import { Button, Tooltip } from "metabase/ui";
+
+export const CommandPaletteTrigger = ({
+  onClick,
+}: {
+  onClick: (e: React.MouseEvent) => void;
+}) => {
+  const METAKEY = isMac() ? "⌘" : "Ctrl";
```

Comment:
!nit I think we should make this a constant somewhere. I see a few places where we do this type of check.